ansible_collections/arista/avd/docs/containers/overview.md

```diff
@@ -0,0 +1,92 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# AVD Dev Containers Overview
+
+> WARNING: AVD Dev Container are currently in the early preview phase and not officially supported for any production cases.
+> They can be still very useful for development and testing. Please report any issues and optimization suggestions.
+
+## Why Dev Containers
+
+Building a functional Ansible environment can be a challenging task. A manually created Ansible environment can be broken due to number of reasons:
```

Comment:
So we mention the benefit of isolating the Ansible environment on the local machine from the environment where development is happening as these might have different requirements etc?

Reply:
done